packages/image_picker/image_picker_ios/lib/image_picker_ios.dart

```diff
@@ -175,6 +175,44 @@ class ImagePickerIOS extends ImagePickerPlatform {
     );
   }
 
+  Future<List<dynamic>> _getMediaPaths({
+    double? maxWidth,
+    double? maxHeight,
+    int? imageQuality,
+    ImagePickerOptions options = const ImagePickerOptions(),
```

Comment:
What about video options like max duration?

Reply:
max duration only applies to captured videos. This method can only pick from the gallery.